src/components/Pong/index.js

```diff
@@ -105,10 +113,17 @@ class Pong extends Component {
           x: ballPosition.x + ballVector.x + paddleVars.deltaX,
           y: ballPosition.y - ballVector.y
         }});
+        const rgbValues = valuesRGB();
+        const colorValues = getRGB(rgbValues);
+        const color = setColor(rgbValues);
+        console.log(color);
+        const luminance = getLuminance(colorValues);
+        const fontColor = getFontColor(luminance);
+        console.log(fontColor);
+        this.props.onChangeColor(color, fontColor);
+        this.props.webrtc.shout('bgColor', { bgColor: color });
+        this.props.webrtc.shout('fontColor', { fontColor: fontColor });
```

Comment:
Don't send fontColor when the recipient can already calculate the luminance from the bgColor they received

Reply:
Fixed it! @lazorfuzz